Vision/Vision.bs

```diff
@@ -142,34 +143,34 @@ Boilerplate: omit conformance
 		and more.
 		In order to ensure the W3C serves the needs of the entire Web user base, 
 		we also strive to broaden diversity and inclusion for our own participants.
-	* We create standards with an ethical intent to improve 
+	* **Horizontal Review**: We create standards with an ethical intent to improve 
 		equity of access and participation on the Web. 
 		We will ensure the technical standards of the Web meets broad goals 
 		using consistent horizontal review to ensure accessibility, 
 		internationalization,
 		sustainability,
 		privacy,
 		and security.
-	* We believe in principled, community-wide consensus-building
+	* **Consensus**: We believe in principled, community-wide consensus-building
 		as the basis for building standards.
-	* Our standards are rooted in a strong royalty-free patent policy
+	* **Free to Implement**: Our standards are rooted in a strong royalty-free patent policy
 		and open copyright licenses.
-	* We welcome individuals and organizations of all sizes
+	* **Open Participation**: We welcome individuals and organizations of all sizes
 		(from single-person companies to multi-nationals),
 		and take feedback from the general public.
-	* We believe proven interoperable implementation is a 
+	* **Interoperability**: We believe proven interoperable implementation is a 
 		requirement for broad adoption of standards. 
 		To ensure reliable interoperability, 
 		we require multiple implementations 
 		and open test suites for our standards.
-	* The Web will continue to expand 
+	* **Incubation**: The Web will continue to expand 
 		in user base, global reach, and technical breadth. 
 		We are committed to encouraging incubation in new areas, 
 		collaborating on innovations across our community.
-	* We aim to reduce centralization in Web architecture,
+	* **Decentralization**: We aim to reduce centralization in Web architecture,
 		minimizing single points of failure
 		and single points of control.
-	* We are committed to establishing and improving collaborative relationships
+	* **Collaboration**: We are committed to establishing and improving collaborative relationships
 		with other Internet and Web standards organizations,
 		and building and maintaining respected relationships
 		with governments and businesses for providing credible advice.
```

Comment:
It's too late in the day for me to remember the grammatical term for this but I wonder if this is more consistent if each principle is an active not passive term.

Diversity >> Diverse
Consensus >> Consensus-Driven
Interoperability >> Interoperable 
Decentralization >> Decentralized
Collaboration >> Collaborative

Reply:
Otherwise, this is exactly what I was looking for!